portal/src/graphql/portal/VerificationConfigurationScreen.tsx

```diff
@@ -127,7 +128,7 @@ function constructConfig(
       delete v.criteria;
     }
 
-    if (a.oob_otp.sms.phone_otp_mode === "sms") {
```

Comment:
Better to define a constant for the default value, it can be a constant in the same file. It is easy to miss if we need to update more than 1 place in the future.